app.py

```diff
@@ -0,0 +1,29 @@
+import flask
+import requests
+import os
+
+app = flask.Flask(__name__)
+
+MY_API_SECRET_KEY = "x95xd2dbxc3x175md5"
+
+BASE_URL = 'https://api.nytimes.com/svc/search/v2/articlesearch.json'
+
+params = {
+    'q': 'braves', # Query keywords
+    'api-key': os.getenv(MY_API_SECRET_KEY),
```

Comment:
Should be using a .env file to call the API Key, the api key will not be fetched because .env does not exist.